star_tides/services/databases/mongo/models/guide_model.py

```diff
@@ -0,0 +1,22 @@
+''' star_tides.services.mongo.models.user_model
+'''
+from mongoengine import (
+    Document,
+    StringField,
+    UUIDField,
+    ListField,
+    EmbeddedDocumentField
+)
+
+from star_tides.services.databases.mongo.models.guidance_model import GuidanceModel
+
+
+class UserModel(Document):
```

Comment:
What is this for? We already have User and Contact models.

Reply:
This was a mistake. I copied UserModel and forgot to update it. Whoops.